docs/api/classifier/class_prediction_error.rst

```diff
@@ -45,7 +45,7 @@ The class prediction error chart provides a way to quickly understand how good y
 API Reference
 -------------
 
-.. automodule:: yellowbrick.classifier.class_balance
+.. automodule:: yellowbrick.classifier.class_prediction_error
```

Comment:
Renamed the module with `ClassPredictionError` and separated it from `ClassBalance`